### PR TITLE
Add sponsor logos, download link on every sidebar

### PR DESCRIPTION
**Pull request by jfrankl**
image

image

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -141,7 +141,7 @@ var nav = new mapboxgl.NavigationControl({
   showZoom: true
 });
 
-mapRegular.addControl(nav, "top-left");
+mapRegular.addControl(nav, "top-right");
```
Comment (rossbernet) [inline js/main.js:144]: I think this is the better option, if for no other reason than it seems standard, but I am curious if you had it on the left initially for a particular reason?

Comment (jfrankl) [reply js/main.js:144]: I don't think I had a specific reason for it being in the top left initially. On Azavea and external mapping projects, I've see the zoom buttons in all corners, and I think any location is fine.

The reason I'm moving it to the top right is to move it away from the "about" button in the sidebar, so there aren't as many distracting buttons in that area. I think this is a bit more clear.

**Comment by jfrankl** [comment]
Made one more small change to make the zoom controls match the dark theme.

image

File: js/main.js (hunk 1 above)
Comment (rossbernet) [reply js/main.js:144]: That makes sense. And thanks for the explanation.

**Comment by jfrankl** [comment]
Let me know if you need anything else before approving.

**Pull request merged**